README.md

```diff
@@ -8,27 +8,29 @@ The original datasets and documents are sourced from the Chicago Police Departme
 
 ### 2. Research Question
 
-With our dataset we would look answer the main inferential question of whether an officer's salary influences their complaint rate (numbe of complaints per year) using a subset of the data from the years 2005 to 2015.
+With our dataset we would look answer the main inferential question of whether an police staff member's salary influences their complaint rate (number of complaints per year) using a subset of the data from the years 2005 to 2015. We suspect a relationship between these two variables 
```

Comment:
Just small grammar change, should be 'a' instead of "whether 'an' police staff member"